specs/IsPinchZoomEnabled.md

```diff
@@ -0,0 +1,71 @@
+# Background
+
+There are two types of zoom in Chromium: Browser Zoom and Pinch-Zoom:
+- Browser zoom is what you get by using Ctrl + +(plus) or – (minus) or Ctrl + Mousewheel. 
+- Pinch-zoom is activated by using a pinch gesture on a touchscreen. 
+
+Currently, the first type of zoom control is supported in WebView2 and modifying it has no effect on pinch zoom. 
+
+In response to customer requests to be able to change the current functionality of pinch zoon in WebView2: Disable/Enanle pinch zoom, the WebView2 team has introduced Pinch Zoom API which allows users to change setting to disable/enable pinch zoom.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+The default value for IsPinchZoomEnabled is true.
+When this setting is set to false, it disables pinch zoom in WebView.
+
+
+# Examples
+```cpp
+void SettingsComponent::TogglePinchZooomEnabled()
+{
+    BOOL enabled;
+    CHECK_FAILURE(coreWebView2Settings->get_IsPinchZoomEnabled(&enabled));
```

Comment:
Would be nice to show where coreWebView2Settings comes from and what coreWebView2Settings is.